standard/sections/clause_7_normative_text.adoc

```diff
@@ -223,7 +223,7 @@ include::../recommendations/core/PER_data_id.adoc[]
 ----
 "properties": {
   ...
-  "data_id": "org1/datasets/123/data/UANT01_CWAO_200445___15103.bufr4"
+  "data_id": "wis2/hkg/hongkong/data/core/weather/surface-based-observations/synop/WIGOS_0-20000-0-45007_20240209T090000"
   ...
```

Comment:
it was pointed out that maybe it's not a good idea to use GTS headers in the examples